integration-cli/docker_cli_run_unix_test.go

```diff
@@ -238,7 +238,7 @@ func (s *DockerSuite) TestRunOOMExitCode(c *check.C) {
 	select {
 	case err := <-errChan:
 		c.Assert(err, check.IsNil)
-	case <-time.After(30 * time.Second):
+	case <-time.After(600 * time.Second):
```

Comment:
@hqhq as far as I remember gccgo(glibc) bootstrap and growing size of the binary was causing the extra time.